Userland/Libraries/LibWeb/Layout/TableFormattingContext.cpp

```diff
@@ -671,6 +675,28 @@ void TableFormattingContext::position_cell_boxes()
     }
 }
 
+CSSPixels TableFormattingContext::compute_row_content_height(Cell const& cell) const
+{
+    auto& row_state = m_state.get(m_rows[cell.row_index].box);
+    if (cell.row_span == 1) {
+        return row_state.content_height();
+    }
+    // The height of a cell is the sum of all spanned rows, as described in
+    // https://www.w3.org/TR/css-tables-3/#bounding-box-assignment
+    CSSPixels span_height = 0;
+    for (size_t i = 0; i < cell.row_span; ++i) {
+        auto const& row_state = m_state.get(m_rows[cell.row_index + i].box);
+        if (i == 0) {
+            span_height += row_state.content_height() + row_state.border_box_bottom();
+        } else if (i == cell.row_span - 1) {
+            span_height += row_state.border_box_top() + row_state.content_height();
+        } else {
+            span_height += row_state.border_box_height();
+        }
+    }
+    return span_height;
+}
```

Comment:
initially, it be difficult to understand why borders are not taken in account when then span is equal to 1, but they are included in content height when span > 1. some additional clarification would be helpful.

Reply:
When the row span is greater than 1, the borders of inner rows within the span have to be included in the content height of the spanning cell. The first top and final bottom are excluded to be consistent with the row span of 1, which excludes the top and bottom border of the single cell (we use the content height in that case).

I'm not sure I got it right, does it make sense to you?

Reply:
yes, that makes sense. if you can put comment that would be nice :)

Reply:
Added a comment.